lib/suspenders.rb

```diff
@@ -1,5 +1,6 @@
 require 'suspenders/version'
 require 'suspenders/generators/app_generator'
+require 'suspenders/generators/static_generator'
```

Comment:
Should these generators be required in `suspenders/generators/app_generator.rb`?